core/scripts/vrfv2/testnet/main.go

```diff
@@ -137,7 +149,7 @@ func main() {
 		coordinatorSub := flag.NewFlagSet("coordinator-subscription", flag.ExitOnError)
 		address := coordinatorSub.String("address", "", "coordinator address")
 		subID := coordinatorSub.Int64("sub", 0, "subID")
-		panicErr(coordinatorSub.Parse(os.Args[2:]))
+		parseArgs(coordinatorSub, os.Args[2:], []string{"address", "pubkey"})
```

Comment:
should this be subID instead of pubkey?

Reply:
Actually @pramodhs2 lets make all sub/subID related flags the same. Looks like they're different depending on which subcommand you invoke.